sos/report/plugins/vdsm.py

```diff
@@ -98,11 +98,12 @@ def setup(self):
                 for pid in qemu_pids
                 for name in files
             ])
-        self.add_cmd_output([
-            "ls -ldZ /etc/vdsm",
-            "su vdsm -s /bin/sh -c 'tree -l /rhev/data-center'",
-            "su vdsm -s /bin/sh -c 'ls -lR /rhev/data-center'"
-        ])
+
+        self.add_dir_listing(
+            ['/etc/vdsm', '/rhev/data-center'],
+            runas='vdsm'
```

Comment:
do we not need tree here for `/rhev/data-center`

Reply:
Oh, good catch!